cpu/atmega256rfr2/include/cpu_conf.h

```diff
@@ -43,7 +43,7 @@ extern "C" {
 #define THREAD_STACKSIZE_DEFAULT   (512)
 #endif
 
-#define THREAD_STACKSIZE_IDLE      (129)
+#define THREAD_STACKSIZE_IDLE      (128)
```

Comment:
This is not an increase? Is this is a typo fix?

Reply:
Oh, yes, should it go in it's own commit do you think?

Reply:
I don't mind, I was just curious